modules/gapi/CMakeLists.txt

```diff
@@ -275,12 +275,19 @@ if(HAVE_GAPI_ONEVPL)
   if(TARGET opencv_test_gapi)
     ocv_target_compile_definitions(opencv_test_gapi PRIVATE -DHAVE_ONEVPL)
     ocv_target_link_libraries(opencv_test_gapi PRIVATE ${VPL_IMPORTED_TARGETS})
+    if(MSVC)
+        target_compile_options(opencv_test_gapi PUBLIC "/wd4201")
+    endif()
     if(HAVE_D3D11 AND HAVE_OPENCL)
         ocv_target_include_directories(opencv_test_gapi SYSTEM PRIVATE ${OPENCL_INCLUDE_DIRS})
     endif()
   endif()
   ocv_target_compile_definitions(${the_module} PRIVATE -DHAVE_ONEVPL)
   ocv_target_link_libraries(${the_module} PRIVATE ${VPL_IMPORTED_TARGETS})
+  target_compile_options(${the_module} PUBLIC "/wd4201"
+                                              "/wd4302"
+                                              "/wd4311"
+                                              "/wd4312")
```

Comment:
I don't think it is good to suppress these warnings for all the module. Is it because of oneVPL headers includes?
Then we should maybe cover it with `pragma push-pop`s, not to affect all the other compiler code check

Reply:
Great!